source/solutions-library/ai-driven-interactive-banking.txt

```diff
@@ -0,0 +1,179 @@
+.. _arch-center-is-interactive-banking:
+
+==========================================
+AI-Driven Interactive Banking
+==========================================
+
+.. meta:: 
+   :keywords: AI, Generative AI, Interactive Banking, Chatbots, MongoDB Atlas, Vector Search, Large Language Models, Financial Services, Personalization.
+   :description: Build an application using MongoDB Atlas Vector Search and large language models to improve the interactivity of banking applications.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+- **Use cases**: `Gen AI <https://www.mongodb.com/solutions/use-cases/artificial-intelligence>`_, 
+  `Personalization <https://www.mongodb.com/solutions/use-cases/personalization>`_
+- **Industries**: `Financial Services <https://www.mongodb.com/solutions/industries/financial-services>`_
+- **Partners**: `Amazon Bedrock <https://aws.amazon.com/bedrock>`_
+
+Solution Overview
+-----------------
+
+Interactive banking represents a new era in financial services where customers engage with
+digital platforms that anticipate, understand, and meet their needs in real time.
+
+This approach uses generative artificial intelligence (gen AI) technologies like chatbots
+and virtual assistants to enhance basic day-to-day banking operations. By leveraging gen
+AI, banks can enable self-service digital channels while simultaneously elevating the
+customer experience through tailored, context-aware interactions. From AI-powered chatbots
+that resolve queries instantly to predictive analytics that offer tailored financial
+advice, interactive banking is no longer just about convenience—it’s about creating a
+smarter, more engaging and more intuitive banking journey for every user.
+
+By integrating AI-driven advisors into the digital banking experience, banks can provide a
+seamless, in-app solution that delivers instant, relevant answers. This removes the need
+for customers to leave the app to sift through pages of bank documentation in search of
+answers, or worse, endure the inconvenience of calling customer service. The result is a
+smoother and more user-friendly interaction, where customers feel supported in their
+self-service journey, free from the frustration of navigating traditional, cumbersome
+information sources. The entire experience remains within the digital space, enhancing
+convenience and efficiency.
+
+.. video:: https://www.youtube.com/watch?v=Pn0IOrn3TKI
+
+Reference Architecture
+-----------------------
+
+The problem with traditional terms and conditions is that they are dense, unstructured,
+and not easily usable within digital banking environments. To remedy this MongoDB and its
+partner propose the following reference architecture:
+
+.. figure:: /includes/images/industry-solutions/leafy-bank-in-aws.svg
+  :width: 750px
+  :height: 750px
```

Comment:
```suggestion
  :width: 750px
```
We only set `:width:`, `:height:` isn't a recognized attribute so it causes build errors